sdk/deviceupdate/azure-iot-deviceupdate/sdk_packaging.toml

```diff
@@ -1,9 +1,6 @@
 [packaging]
 auto_update = false
 package_name = "azure-iot-deviceupdate"
-package_pprint_name = "Azure Device Update"
+package_pprint_name = "Azure DeviceUpdate"
```

Comment:
Should we change the pprint name? @annatisch

Reply:
agree with you. I think we had better not change the pprint name.